### ISSUE TITLE
Drip topic date Calendar widget does not work properly

### ISSUE THREAD

**Issue by onno-schuit** [solin-repo/moodle-format_drip#1]
The calendar widget does not work properly. The layout is very minimal. If you try to select a different date after selecting a date for the first time, the widget seems to crash.

The widget does not disappear, remains stuck on the screen until you refresh the page.

![image](https://user-images.githubusercontent.com/105123/96024077-93391c00-0e53-11eb-9763-2da64b3d07ab.png)

**Comment by denisjuergen** [comment]
@onno-schuit in what Moodle version did you notice this bug? I tried reproducing this issue on 3.9 (2020061500) and couldn't find it.
I noticed the plugin requires [Moodle 3.9 rc-1 (2020060900)](https://github.com/moodle/moodle/blob/v3.9.0-rc1/version.php), which might present some bugs (given it is a release candidate); maybe we should consider incrementing the required version to 2020061500, which is for the stable 3.9 release.
